src/frontend/components/SøknadsSteg/OmBarnet/useOmBarnet.tsx

```diff
@@ -253,19 +257,20 @@ export const useOmBarnet = (
         fjernPeriode: fjernBarnetrygdsperiode,
         leggTilPeriode: leggTilBarnetrygdsperiode,
         registrertePerioder: registrerteEøsBarnetrygdsperioder,
-    } = usePerioder<IEøsBarnetrygdsperiode>(
-        gjeldendeBarn.eøsBarnetrygdsperioder,
-        { mottarEllerMottokEøsBarnetrygd },
-        avhengigheter => avhengigheter.mottarEllerMottokEøsBarnetrygd.verdi === ESvar.JA,
-
-        (felt, avhengigheter) => {
+    } = usePerioder<IEøsBarnetrygdsperiode>({
+        feltId: `${BarnetrygdperiodeSpørsmålId.barnetrygdsperiodeEøs}-${PersonType.Søker}`,
+        verdi: gjeldendeBarn.eøsBarnetrygdsperioder,
```

Comment:
Litt usikker på om denne PersonTypen burde være .Søker, skal jeg legge inn en ny PersonType som er barn?

Reply:
Den må være unik, så da vil det ikke funke med barn fordi da får man fortsatt to like id'er; eøsBarnetrygdperiode-barn -> dersom man har to barn. Så her kunne man feks brukt id'en til barnet 😊

Reply:
Når jeg tenker meg om vil jo dette gjelde alle! arbeidsPeriodeNorge-andreForelder vil kunne finnes flere ganger hvis det er flere barn, kanskje man bør slenge på barnets id på egentlig alle?

Reply:
`${SpørsmålId}-${PersonType}-${BarnetsId}`

Reply:
Da er persontype lagt til

Reply:
og barnetsid